pkg/resizer/trivial_resizer.go

```diff
@@ -0,0 +1,37 @@
+/*
+Copyright 2018 The Kubernetes Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package resizer
+
+import (
+	"k8s.io/api/core/v1"
+	"k8s.io/apimachinery/pkg/api/resource"
+)
+
+// newTrivialResizer returns a trivial resizer which will mark all pvs' resize process as finished.
+func newTrivialResizer() Resizer {
+	return &trivialResizer{}
+}
+
+type trivialResizer struct{}
+
+func (r *trivialResizer) CanSupport(pv *v1.PersistentVolume) bool {
+	return true
+}
+
+func (r *trivialResizer) Resize(pv *v1.PersistentVolume, requestSize resource.Quantity) (newSize resource.Quantity, fsResizeRequired bool, err error) {
+	return requestSize, false, nil
+}
```

Comment:
so this is a placeholder file that will eventually call out to CSI plugin right?

Reply:
Yeah, I intend to implement the CSI plugin in subsequent PRs.